src/main.ts

```diff
@@ -25,7 +25,7 @@ export default async function(api: (token: string) => GitHub): Promise<void> {
   const externalToken = process.env.COMMITTER_TOKEN || ''
 
   const [owner, repo] = getInput('homebrew-tap', { required: true }).split('/')
-  const formulaName = getInput('formula-name', { required: true })
+  const formulaName = getInput('formula-name') || context.repo.repo.toLowerCase()
```

Comment:
We actually have two ways to get this:

we could do `const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/')` ourselves; which is exactly what octokit does internally.

Or we can go through octokit, which provides an _additional_ fallback leveraging the payload. I prefer this, especially because this will also raise a helpful error if it's unsuccessful. Bummer that the getter is named `repo` and the repository name property itself is _also_ named `repo`.